packages/ag-charts-community/src/scene/shape/text.ts

```diff
@@ -699,5 +716,5 @@ function getVerticalOffset(textBaseline: CanvasTextBaseline): number {
 }
 
 export function splitText(text?: string) {
-    return typeof text === 'string' ? text.split(/\r?\n/g) : [];
+    return typeof text === 'string' ? text.split('\n') : [];
```

Comment:
Does this case deal with Windows carriage return cases correctly?

Reply:
Since we can't really test this on a Linux machine, I suggest we merge and then ask David to test this on a Windows machine.